app/assets/javascripts/i18n.js

```diff
@@ -497,6 +510,63 @@
     return message.replace(/_#\$#_/g, "$");
   };
 
+  // Not part of the public API
+  I18n.splitInterpolate = function(message, matches, options) {
+    var value
+    , placeholder
+    , name
+    , regex;
+
+    var buffer = [message];
+
+    while (matches.length) {
+      placeholder = matches.shift();
+      name = placeholder.replace(this.placeholder, "$1");
+
+      if (this.isSet(options[name])) {
+        value = options[name];
+      } else if (name in options) {
+        value = this.nullPlaceholder(placeholder, message, options);
+      } else {
+        value = this.missingPlaceholder(placeholder, message, options);
+      }
```

Comment:
True, they seem to do the same thing. However, that's how it was in the original implementation. I don't want to change it, because if I end up doing a PR to the upstream repository, I think it's not good to include too many refactorings in the PR.